src/interfaces.ts

```diff
@@ -123,10 +118,25 @@ export interface Connection {
   contract: string;
   list: () => Promise<TableMetadata[]>;
   create: (
-    query: string,
-    options?: CreateTableOptions
+    /** The ChainId of the chain this table should be created on.
+     *  MUST match the chain that the SDK connected to
+     **/
+    chainId: number,
+    /** The schema that defines the columns and constraints of the table,
+     *  e.g.
+     * `id int NOT NULL,
+     *  name char(50) NOT NULL,
+     *  favorite_food char(50),
+     *  PRIMARY KEY (id)`
+     */
+    schema: string,
+    /** an optional prefix to the tablename that will be assigned to this table.
+     *  If supplied, it must conform to the rules of SQL table names
+     **/
+    prefix?: string
   ) => Promise<ContractReceipt>;
-  query: (query: string) => Promise<null | ReadQueryResult>;
+  read: (query: string) => Promise<null | ReadQueryResult>;
```

Comment:
Why would null show up? I'd rather not have `null` in our public APIs, as this is frowned upon in typescript. It would be better to always be a ReadQueryResult or error if something goes wrong.